packages/transport/src/modules/tags/backbone/BackboneGetTag.ts

```diff
@@ -0,0 +1,9 @@
+export interface BackboneAttributeTagCollection {
```

Comment:
Can we omit the `Backbone` here in the name and simply call it `AttributeTagCollection`? Same below. And same for the function name of the Client.

Reply:
We have that for all the types coming from the backbone